content/blog/2025/oddcontrast-guide.md

```diff
@@ -0,0 +1,130 @@
+right arrow icon button or you can drag and drop the color swatches themselves.
+
+@@@TODO add BG/FG swap image
+
+Once you are happy with your colors, you can easily copy your preferred color
+notation to the clipboard with a single click.
+
+@@@TODO add copy to clipboard image
+
+
+## Visualize the in-gamut range
+
+OddContrast allows you to choose whether or not to display a color gamut for
+visual reference, and if so, which specific gamut you'd like to see. Once a
+gamut is selected, the color sliders will display the "in-gamut" boundaries and
+will update with the color values. To preview this, select the sRGB option in
+the "Show Gamut" field, edit the color sliders and see how every shift in color
+will affect the other channels.
+
+A warning will be displayed if your color is out of gamut for each of the color
+spaces, regardless of your gamut display preference.
+
+## Saving and sharing your color combinations
+
+OddContrast automatically updates the URL as you make changes to the colors.
+This dynamic URL allows you to easily share color combinations with others or
+bookmark them for your own future use.
+
+## For fun
+
+To add some whimsy, OddContrast proudly showcases your chosen color
+combination within its very own logo.
+
+## Learn more about the available colors spaces and formats
+
+- [Okay, Color Spaces](https://ericportis.com/posts/2024/okay-color-spaces/) by Eric Portis
+- [LCH colors in CSS: what, why, and how?](https://lea.verou.me/blog/2020/04/lch-colors-in-css-what-why-and-how/) by Lea Verou
+- [Falling For Oklch: A Love Story Of Color Spaces, Gamuts, And CSS](https://www.smashingmagazine.com/2023/08/oklch-color-spaces-gamuts-css/) by Geoff Graham
+- [A perceptual color space for image processing](https://bottosson.github.io/posts/oklab/) by Björn Ottosson
+- [OKLCH in CSS: why we moved from RGB and HSL](https://evilmartians.com/chronicles/oklch-in-css-why-quit-rgb-hsl) by Evil Martians
+
+## ToDo / Things to include
+
+- [ ] Evaluates the contrast ratio between two colors
+- [ ] WCAG 2 Level AA and AAA pass/fail results for both normal and large text
+- [x] Resources section that has links to learn more about color
+- [x] Shows the selected colors in all the supported color notation
+- [x] Indicates when a chosen color falls outside the current color gamut
+
+
+## Background that I probably don't need, was going to start with this
+
+When HSL was introduced in the
+[CSS Color Module 3](https://www.w3.org/TR/css-color-3/#hsl-color)
+it offered an intuitive approach to color manipulation.
+Designers and developers could more easily predict the resulting
+color when adjusting the Hue (H), Saturation (S), and Lightness (L) values.
+Now with the [Color 4 specification](https://www.w3.org/TR/css-color-4/),
+color formats like Oklch offer similar predictability but with an even broader
+palette of possibilities as it isn't confined to the sRGB color space.
```

Comment:
This is that chunk of color info that I started to write about but thought this would be better as a completely separate post but also would make any post extremely long.